eng/pipelines/templates/CrossFramework_Tests_On_Windows.yml

```diff
@@ -0,0 +1,75 @@
+steps:
+- task: PowerShell@2
+  displayName: "Print Environment Variables"
+  inputs:
+    targetType: "inline"
+    script: |
+      Write-Host "##vso[build.updatebuildnumber]$env:FullVstsBuildNumber"
+      Get-ChildItem Env: | Sort-Object Name | Format-Table -Wrap -AutoSize
+
+- task: PowerShell@1
+  displayName: "Run Configure.ps1"
+  inputs:
+    scriptName: "$(Build.Repository.LocalPath)\\configure.ps1"
+    arguments: "-Force -CleanCache"
+
+- task: MSBuild@1
+  displayName: "Restore for VS2019"
+  inputs:
+    solution: "build\\build.proj"
+    configuration: "$(BuildConfiguration)"
+    msbuildArguments: "/t:RestoreVS /p:BuildNumber=$(BuildNumber) /p:BuildRTM=false /v:m"
+
+- task: MSBuild@1
+  displayName: "Run Cross Verify Tests (continue on error)"
+  continueOnError: "true"
+  inputs:
+    solution: "build\\build.proj"
+    configuration: "$(BuildConfiguration)"
+    msbuildArguments: "/t:CrossVerifyTests /p:BuildRTM=false /p:BuildNumber=$(BuildNumber) /p:TestResultOutputFormat=xml"
```

Comment:
I just tried to run:
 ```
.\cli\dotnet.exe test .\test\NuGet.Core.FuncTests\NuGet.Signing.CrossFramework.Test\bin\Debug\netcoreapp5.0\NuGet.Signing.CrossFramework.Test.dll
```
before and after I delete the VS15 folder under artifacts.
When VS15 folder exists, all the 8 tests passed.
When VS15 folder is deleted, all the 8 tests failed for:
```
Error Message:
   System.ComponentModel.Win32Exception : The system cannot find the file specified.
  Stack Trace:
     at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at NuGet.Test.Utility.CommandRunner.Run(String process, String workingDirectory, String arguments, Boolean waitForExit, Int32 timeOutInMilliseconds, Action`1 inputAction, IDictionary`2 environmentVariables) in C:\repos\NuGet.Client\test\TestUtilities\Test.Utility\CommandRunner.cs:line 68
   at NuGet.Signing.CrossFramework.Test.CrossFrameworkVerificationTest.RunVerifyCommand(String packagePath) in C:\repos\NuGet.Client\test\NuGet.Core.FuncTests\NuGet.Signing.CrossFramework.Test\CrossFrameworkVerificationTest.cs:line 401
   at NuGet.Signing.CrossFramework.Test.CrossFrameworkVerificationTest.Verify_AuthorSignedTimestampedRepositoryCounterSignedTimestampedPackage_SuccessAsync() in C:\repos\NuGet.Client\test\NuGet.Core.FuncTests\NuGet.Signing.CrossFramework.Test\CrossFrameworkVerificationTest.cs:line 267
```

Reply:
The reason why cross verify tests on CI passed without downloading the artifacts folder before hand is, there is a "Run Configure.ps1" in eng\pipelines\templates\CrossFramework_Tests_On_Windows.yml. So it creates artifacts itself.

Reply:
Good catch! The DownloadBuildArtifacts task is removed.

Reply:
For now, it's working like unit tests in Build_and_UnitTest if using our build\build.proj infrastructure.
So for the net472 test assembly, [it will be tested by xunit.console.x86.exe](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4997188&view=logs&j=86755a60-c91a-5662-e27a-82f2078d7d0a&t=b2085de5-83eb-5eb7-601c-4200a643c07d&l=37993).
For the net5.0 test assembly, [it will be tested by dotnet.exe test](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4997188&view=logs&j=86755a60-c91a-5662-e27a-82f2078d7d0a&t=b2085de5-83eb-5eb7-601c-4200a643c07d&l=38024).

I tried to run dotnet.exe test to run both net472 and net5.0 test assemblies, both of them passed successfully.
Do you think I need to change the build.proj to use dotnet.exe test to run all test assemblies? Or just make it work for cross framework verification tests? Thanks!

Reply:
> Do you think I need to change the build.proj to use dotnet.exe test to run all test assemblies?

I would like us to yes, but when I tried a while ago, a bunch of tests failed, so unless you want to spend the effort to do so now, no, don't do it for this PR.

> Or just make it work for cross framework verification tests?

Have separate build and test jobs.  We can have `msbuild build\build.proj -t:BuildTests` or whatever, but don't run the tests. Once the binaries are built, have a separate `dotnet test test\whatever\ --no-build`.

Reply:
Thanks! Changed to use `dotnet test` to run the cross verify tests.